mlos_bench/mlos_bench/storage/sql/common.py

```diff
@@ -4,15 +4,49 @@
 #
 """Common SQL methods for accessing the stored benchmark data."""
 
+from typing import Any
+
 import pandas
 from sqlalchemy import Integer, and_, func, select
-from sqlalchemy.engine import Engine
+from sqlalchemy.engine import Connection, Engine
+from sqlalchemy.schema import Table
 
 from mlos_bench.environments.status import Status
 from mlos_bench.storage.base_experiment_data import ExperimentData
 from mlos_bench.storage.base_trial_data import TrialData
 from mlos_bench.storage.sql.schema import DbSchema
-from mlos_bench.util import utcify_nullable_timestamp, utcify_timestamp
+from mlos_bench.util import nullable, utcify_nullable_timestamp, utcify_timestamp
+
+
+def save_params(
```

Comment:
Originally refactored for a different trial_runner_id schema method, but might still be useful to leave?
Maybe just split out to a separate PR?

Reply:
I think we can leave it in this PR. it all looks good together

Reply:
I might add some tests then. I don't think add_new_config_params is used anymore.